tests/Operation/AggregateFunctionalTest.php

```diff
@@ -274,12 +277,50 @@ public function provideTypeMapOptionsAndExpectedDocuments()
         ];
     }
 
+    public function testReadPreferenceWithinTransaction()
+    {
+        $this->skipIfTransactionsAreNotSupported();
+
+        // Collection must be created before the transaction starts
+        $options = ['writeConcern' => new WriteConcern(WriteConcern::MAJORITY)];
+        $operation = new CreateCollection($this->getDatabaseName(), $this->getCollectionName(), $options);
+        $operation->execute($this->getPrimaryServer());
```

Comment:
These three lines are copied many times, perhaps they should go into a helper function in FunctionalTestCase?

Reply:
Good idea. I'll make the change and update the PR.